cloudbrain/rt_server/rt_server.py

```diff
@@ -14,6 +15,7 @@
 
 SERVER_PORT = 31415
 logging.getLogger().setLevel(logging.INFO)
+recursivedict = lambda: defaultdict(recursivedict)
```

Comment:
Please leave 3 blank lines between static methods, import block, and variables.

So:
```
 from tornado.web import RequestHandler, Application
  


  SERVER_PORT = 31415
  _LOGGER = logging.getLogger().setLevel(logging.INFO)


recursivedict = lambda: defaultdict(recursivedict)


def get_arg_parser():
```